src/views/idp-discovery/IDPDiscoveryForm.js

```diff
@@ -25,13 +25,7 @@ define([
     saveId: 'idp-discovery-submit',
 
     initialize: function () {
-      this.listenTo(this, 'save', function () {
-        var creds = {
-          username: this.model.get('username')
-        };
-        this.settings.processCreds(creds)
```

Comment:
I thought that the point of this was that the widget consumer can modify the username before it gets saved. Is that not the case anymore? When does `processCreds` gets called then?

Reply:
This is where monolith provides the hook.
https://github.com/okta/okta-core/blob/master/clients/loginpage/src/main-loginpage.js#L127-L145

I think the username/password are pass as value (read-only) to `processCreds` hook hence it's supposed not to be modified.

Reply:
Even if it's read only, where would be the place customers can hook on to if they host the widget themselves? Since this is idp discovery, isn't a call back to processCreds expected with just the username?

Reply:
@santhoshbalakrishnan 
Good question. AFAIK, there are no use cases today that customers would run some callback logic only on username at IdP discovery login flow.
ping @yuliu-okta @petermiller-okta to confirm.

Reply:
`processCreds` still being used at Primary Auth flow.

Reply:
According to the widget doc, `processCreds` is to handle credentials and should be only called in Primary Auth, Password Expiration, and Password Reset flows. If the identifier first flow needs a callback in the future, we might need to provide a new callback instead of `processCreds` since it is not credentials.

Reply:
I think this makes sense. If Okta is the IdP, then we will go to PrimaryAuth and there we will be able to send username/password to the callback, which will do the logic for chromeOs. 
If Okta is not the IdP, then I don't know how this works, but not sure that is supported

A question I have is when we have passwordless, then I don't think we can support chromeOS login the way it is right now...

Reply:
@mauriciocastillosilva-okta 
yeah, I believe it won't work when we have passwordless. who's the PMs owning passwordless? we might want to give heads up in case they're not aware yet.

Reply:
I will let him know, since I've been working in that feature.